.github/workflows/protobuf.yml

```
@@ -32,10 +32,9 @@ jobs:
           fi
         # yamllint enable
 
-  # add this back when we start using versioning
-  # breakage:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - name: check-breakage
-  #       run: make proto-check-breaking
+  breakage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: check-breakage
```

[nit] can the step name match the Makefile task name so it's easy for unfamiliar maintainers to connect the GH action in the list of CI tasks to the Makefile task? Note, the suggestion is consistent with proto-lint and proto-gen above.

```suggestion
  proto-check-breaking:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - name: proto-check-breaking
```